papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -114,4 +144,23 @@ private Sprint getValidSprint(Long sprintId) {
         return sprintService.getById(sprintId)
                 .orElseThrow(ResourceNotFoundException::new);
     }
+
+    public ResponseEntity<SprintSummaryDto> getSprintSummary(Long projectId, Long sprintId) {
+        Sprint sprint = getValidSprint(sprintId);
+        Project project = projectsRestService.getValidProject(projectId);
+
+        if (project.isAdmin(userContext.getUser())) {
+            throw new ForbiddenAccessException();
+        }
+
+        List<UserAvailabilityDto> userAvailabilityDtos = availabilityRestService.getUserAvailabilities(sprintId);
+        Duration totalAvailableTime = availabilityRestService.getTotalAvailableTimeBySprintId(sprintId);
+
+        double coefficient = sprint.getTimePlanned().dividedBy(sprint.getTimeBurned());
```

Comment:
To na razie dummy kod, żeby zmiany znalazły się szybciej na masterze właściwa logika wskoczy w nowym tasku. Nie dodawałem wywołania jakieś dummy metody, żeby nie wprowadzać dyskusji o tym gdzie ją dodać czy może w Sprint czy może tutaj i tak dalej

Reply:
W sumie można zostawić komentarz w kodzie o tym, że to jest tymczasowa uproszczona implementacja i "to be expanded in issue IO2019TEAM-100" czy coś

Reply:
Dodano informacje